src/main/java/com/squareup/javawriter/ClassName.java

```diff
@@ -103,42 +102,6 @@ public String canonicalName() {
     return fullyQualifiedName;
   }
 
-  public String classFileName() {
```

Comment:
Simply, I didn't think we needed 'em for JavaWriter. This is a weird method anyway, since it doesn't include the `.class` suffix that might be expected.

What does Dagger use this for? Would it make more sense to define it where it's needed?